autonomy/cli/scaffold_fsm.py

```diff
@@ -81,6 +81,9 @@
 BEHAVIOURS_FILENAME = "behaviours.py"
 MODELS_FILENAME = "models.py"
 HANDLERS_FILENAME = "handlers.py"
+DIALOGUES_FILENAME = "dialogues.py"
+PAYLOADS_FILENAME = "payloads.py"
```

Comment:
I would like to scaffold the payloads and dialogues as well in two subsequent PRs after `test_rounds.py` and `test_behaviours.py` are covered

Reply:
Good idea, but remove it from this PR so it's self-contained.